calcom/cal.com
# Add helpscout to billing page

## What does this PR do?



Replaces Intercom with Helpscout on the billing page

Fixes # (issue)



https://www.loom.com/share/89bca93871394c71914bb3f8cf096994

**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Go to /settings/billing
- Click "Contact our support team"
	-  Helpscout window should popup
	- Note you need a Helpscout key

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
